Gruntfile.js

```diff
@@ -77,7 +77,7 @@ module.exports = function (grunt) {
    },
    copy: {
     files: ['Gruntfile.js', 'js_src/rdm.js'],
-    tasks: ['clean:copy', 'copy:develop'],
+    tasks: ['jshint:dev', 'clean:copy', 'copy:develop'],
```

Comment:
I added jshint here because it's not worth copying the files if they don't parse correctly.